pkg/kubelet/server/metrics/metrics.go

```diff
@@ -40,7 +40,7 @@ var (
 		// server_type aims to differentiate the readonly server and the readwrite server.
 		// long_running marks whether the request is long-running or not.
 		// Currently, long-running requests include exec/attach/portforward/debug.
-		[]string{"method", "path", "server_type", "long_running"},
+		[]string{"method", "path", "server_type", "long_running", "http_response_code"},
```

Comment:
cc @logicalhan on metrics change

Reply:
Two things:

1. this is a breaking change to the metric, so we will need SIG node to sign off on this. (@dashpole, @SergeyKanzhelev) 
2. conventially, we use `code` elsewhere for the label name for http status codes (https://github.com/kubernetes/kubernetes/blob/17312ea4a92a0bba31272a6709b37a88aa383b2d/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L86).